docs/app/e2e/app.scenario.js

```diff
@@ -39,7 +39,7 @@ describe('docs.angularjs.org', function () {
 
 
     it('should change the page content when clicking a link to a service', function () {
-      browser.get('build/docs/index.html');
+      browser.get('build/docs/index-production.html');
```

Comment:
So, by using `-production` we are not testing against our local build, but only against the CDN version, right?
This makes sense, since it is a separate app that just happens to live on the same repo and the app will use a CDN version (not our local build).